src/engine/strat_engine/cmd.rs

```diff
@@ -456,7 +456,7 @@ pub fn thin_metadata_size(
         .read_to_string(&mut output)?;
     if is_ok {
         Ok(min(
-            Sectors(
+            8u64 * Sectors(
```

Comment:
We had better make this a constant and document it as best we can, even though there's not much to say.

Reply:
Resolved.